packages/editor-ui/src/components/CredentialPicker/CredentialPicker.vue

```diff
@@ -0,0 +1,132 @@
+<template>
```

Comment:
Nitpick, but we started writing `<template>` after `<script setup>` in out components. I think we should stick to that standard.

Reply:
Check! I was actually wondering which is the correct way to go with, as I've seen both. Will change!